manifests/p/prefix-dev/pixi/0.9.1/prefix-dev.pixi.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created with Komac v1.9.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: prefix-dev.pixi

```

Comment:
I guess `dev.prefix.pixi` is not the format that is used here, right?

Reply:
I like all lowercase better

Reply:
another data point, the copyright notice in License is lower case.